x-pack/plugins/cloud_security_posture/server/routes/configuration/update_rules_configuration.ts

```diff
@@ -68,16 +68,22 @@ export const getCspRules = (
 export const createRulesConfig = (
   cspRules: SavedObjectsFindResponse<CspRule>
 ): CspRulesConfiguration => {
-  const activatedRules = cspRules.saved_objects.filter((cspRule) => cspRule.attributes.enabled);
+  const benchmarkType = cspRules.saved_objects[0]?.attributes.metadata.benchmark.id;
+
+  if (!benchmarkType) throw new Error('Expected benchmark type to exists');
```

Comment:
`cspRules` can be an empty array and we should not throw an error in that case
This is a valid situation

Reply:
1. do you mean when users manually delete the saved-object rules? 
2. we need to extract `benchmarkType` from the rules and not from the package policy ( as in - `getBenchmarkInputType(packagePolicy.inputs))` because only that `benchmark.id` is gaurnteed to be the same one used to create the rules. if users change the inputs, the value we extract would be different but the rules were already created, and we don't do anything in `packagePolicyUpdate`